plugins/cfhour.js

```diff
@@ -85,9 +87,9 @@ module.exports = (function(){
 				}
 				else {
 					console.error(error);
+					onErrorDebounce(error);
```

Comment:
This line needs a 2nd argument, right? something like `onErrorDebounce('parsing rss', error);`